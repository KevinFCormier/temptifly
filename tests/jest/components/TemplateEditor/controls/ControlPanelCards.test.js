'use strict'

import React from 'react'
import ControlPanelCards from '../../../../../src/controls/ControlPanelCards'
import renderer from 'react-test-renderer'

import { control } from '../../TestingData'

describe('ControlPanelCards component', () => {
  it('renders as expected', () => {
    const fn = jest.fn()
    const component = renderer.create(
      <ControlPanelCards
        key={'key'}
        control={control}
        controlId={'controlId'}
        handleChange={fn}
        i18n={fn}
      />
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
