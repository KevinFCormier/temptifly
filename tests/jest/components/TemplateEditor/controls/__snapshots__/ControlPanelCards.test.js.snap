// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ControlPanelCards component renders as expected 1`] = `
<div
  className="creation-view-controls-card-container"
>
  <div
    className="tf--grid-container"
  >
    <div
      className="tf--grid"
    >
      <div
        className="tf--providers-container tf--row"
      />
    </div>
  </div>
</div>
`;
